src/rpc/util.cpp

```diff
@@ -493,6 +493,14 @@ RPCHelpMan::RPCHelpMan(std::string name, std::string description, std::vector<RP
         for (const std::string& name : names) {
             CHECK_NONFATAL(named_args.insert(name).second);
         }
+        if (arg.m_type == RPCArg::Type::OBJ_NAMED_PARAMS) {
+            for (const auto& inner : arg.m_inner) {
+                std::vector<std::string> inner_names = SplitString(inner.m_names, '|');
+                for (const std::string& inner_name : inner_names) {
+                    CHECK_NONFATAL(named_args.insert(inner_name).second || inner.m_opts.also_positional);
```

Comment:
I think this should be:

```c++
if (!inner.m_opts.also_positional) {
    CHECK_NONFATAL(named_args.insert(inner_name).second);
}
```

Otherwise a duplicate positional parameter after the `OBJ_NAMED_PARAMS` entry will give an error.